giza/giza/content/options/tasks.py

```diff
@@ -44,8 +44,16 @@ def option_tasks(conf):
 
     tasks = []
     for dep_fn, option in o.content_iter():
+        if option.has_field('command'):
+            t = option.program.split(" ")
```

Comment:
It has occurred to me that I can just string replace any spaces here with dashes for the filename, so I've done that instead of the conditions :)